app/containers/Root/routesConfig.js

```diff
@@ -18,11 +19,15 @@ export const routes = [
     component: Company,
   },
   {
-    path:'/management',
+    path: '/management',
     component: StudentManagement
   },
   {
     path: '/posts',
     component: InternshipPosts,
+  },
+  {
+    path: "/studentsdashboard",
```

Comment:
This route must be removed, you should use the HomePage container that is modified on development.